languages/en.ini

```diff
@@ -953,6 +953,8 @@ page_last = "Go to Last Page"
 page_next = "Go to Next Page"
 page_num = "Page %%page%%"
 page_prev = "Go to Previous Page"
+page_reload_on_deselect_hint = "Page will reload when a filter removed."
```

Comment:
missing "is" -- should be "when a filter is removed."

Reply:
Good find!